kubeflow/pytorch-job/tests/pytorch-job_test.jsonnet

```diff
@@ -224,6 +224,4 @@ local testSuite = suiteResult {
   testCases: testCasesWithResults,
 };
 
-// TODO(https://github.com/kubeflow/kubeflow/issues/1988): Output testSuite
-// rather than std.assertEqual
-std.assertEqual(testSuite.pass, true)
+testSuite.pass
```

Comment:
should be testSuite only?
Since you are getting the "pass" key in python.

Reply:
You're right thanks.